koans/about_tuples.py

```diff
@@ -47,7 +47,7 @@ def test_tuple_constructor_can_be_surprising(self):
 
     def test_creating_empty_tuples(self):
         self.assertEqual(__ , ())
-        self.assertEqual(__ , tuple()) #Sometimes less confusing
+        self.assertEqual(__ , tuple()) # Sometimes less confusing
```

Comment:
The Flake8/PEP 8 guidelines suggest two spaces before, one after. 
```suggestion
        self.assertEqual(__ , tuple())  # Sometimes less confusing
```